package.json

```diff
@@ -34,6 +34,19 @@
     "open formula",
     "openformula"
   ],
+  "browserslist": [
+    "maintained node versions",
```

Comment:
I think `maintained node versions` should be removed, because the `dist` packages are not supposed to be used in Node.

Reply:
Sure. It's gone ;)